block-storage/src/main/scala/coop/rchain/blockstorage/FileLMDBIndexBlockStore.scala

```diff
@@ -131,7 +136,7 @@ class FileLMDBIndexBlockStore[F[_]: Monad: Sync: RaiseIOError: Log] private (
         result <- filteredIndex.flatTraverse {
                    case (blockHash, indexEntry) =>
                      readBlockMessage(indexEntry)
-                       .map(block => List(blockHash -> block))
+                       .map(block => List(blockHash -> BlockMessage.from(block).right.get)) // TODO FIX-ME
```

Comment:
BlockStore works internally on protobufs so each message it contains will be in proper format (this code will only break if it wsa broken before). Nevertheless this should be address in the future (this PR exposed the problem that wsa not visible before - what if protobufs stored in the past are no longer valid objects in the future)

Reply:
please open a ticket for this TODO

Reply:
Done https://rchain.atlassian.net/browse/RCHAIN-3844